### PR TITLE
Sign up/login page with user context wrapped around the App.js

### PR DESCRIPTION
**Pull request by abid-salahi**

### CODE REVIEW

File: src/components/Login/GoogleLoginBtn.js
Comment (jacquelineyin) [inline src/components/Login/GoogleLoginBtn.js]: Should we put this in an env file or something? :O

File: src/components/Login/GoogleLoginBtn.js
Comment (jacquelineyin) [inline src/components/Login/GoogleLoginBtn.js]: Suggest changing it to "isLoggedIn" ("Login" is a noun, so Logined doesn't really work >.<)

File: src/components/Login/GoogleLoginBtn.js
Comment (jacquelineyin) [inline src/components/Login/GoogleLoginBtn.js]: Suggest removing console.log ;D

File: src/components/Login/GoogleLoginBtn.js
Comment (jacquelineyin) [inline src/components/Login/GoogleLoginBtn.js]: Suggest removing console.log ;D

**Review by jenessatan — APPROVED** [review]
Looks good to me. Though I'll let the others take a look too

File: src/components/Login/GoogleLogoutBtn.js
Comment (jacquelineyin) [inline src/components/Login/GoogleLogoutBtn.js]: .env file?

**Review by jacquelineyin — APPROVED** [review]
Looks pretty good. We might want to consider moving the client_id outside of our code in the future. 

Also, because user is now tied to the backend, do we need to have the backend running in order to have things working? (I ask because makeOffer requires the currentUser info)

File: src/components/Login/GoogleLoginBtn.js
Comment (abid-salahi) [reply src/components/Login/GoogleLoginBtn.js]: Yes changes will be made. Do not merge yet. Still integrating user profile page with login.

**Comment by abid-salahi** [comment]
Yes backend needs to run at the same time since the backend authenticates and registers the user info

**Comment by abid-salahi** [comment]
I will merge the pr later once i have user profile integrated with login

**Pull request merged**
